contracts/modules/grouping/EvenSplitGroupPool.sol

```diff
@@ -195,29 +181,22 @@ contract EvenSplitGroupPool is IGroupRewardPool, ProtocolPausableUpgradeable, UU
         address[] memory ipIds
     ) internal returns (uint256[] memory rewards) {
         rewards = _getAvailableReward(groupId, token, ipIds);
+        uint256 totalRewards = 0;
```

Comment:
`_distributeRewards` is only called once so it may not need to be an internal function

Reply:
merged into the external function distributeReward().